Fix build errors when using different javac then jamvm

like

| source/javax/activation/MimeTypeParameterList.java:128: error: cannot find symbol
|         MimeType.checkValidity(name, "Parameter name is invalid");
|                 ^
|   symbol:   method checkValidity(String,String)
|   location: class MimeType
| source/javax/activation/MimeType.java:262: error: no suitable constructor found for MimeTypeParseException(String,String)
|         throw new MimeTypeParseException(message, token);
|               ^


Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Index: activation-1.1.1/source/javax/activation/MimeType.java
===================================================================
--- activation-1.1.1.orig/source/javax/activation/MimeType.java	2006-05-13 02:50:11.000000000 -0700
+++ activation-1.1.1/source/javax/activation/MimeType.java	2013-05-23 21:20:33.181013888 -0700
@@ -253,7 +253,7 @@
     checkValidity(subType, "Sub type is invalid");
   }
 
-  static void checkValidity(String token, String message)
+  public static void checkValidity(String token, String message)
     throws MimeTypeParseException
   {
     int len = token.length();
@@ -271,7 +271,7 @@
       }
   }
   
-  static boolean isValidChar(char c)
+  public static boolean isValidChar(char c)
   {
     return c > ' ' && c <= '~' && TSPECIALS.indexOf(c) == -1;
   }
Index: activation-1.1.1/source/javax/activation/MimeTypeParseException.java
===================================================================
--- activation-1.1.1.orig/source/javax/activation/MimeTypeParseException.java	2006-05-13 02:50:11.000000000 -0700
+++ activation-1.1.1/source/javax/activation/MimeTypeParseException.java	2013-05-23 21:21:14.745013916 -0700
@@ -58,7 +58,7 @@
    * @param message the exception message
    * @param token the token in error
    */
-  MimeTypeParseException(String message, String token)
+  public MimeTypeParseException(String message, String token)
   {
     this(new StringBuffer(message)
          .append(':')
